app/forms/delete_user_email_form.rb

```diff
@@ -48,5 +48,7 @@ def notify_subscribers
     return unless IdentityConfig.store.push_notifications_enabled
     event = PushNotification::IdentifierRecycledEvent.new(user: user, email: email_address)
     PushNotification::HttpPush.deliver(event)
+    event = PushNotification::EmailChangedEvent.new(user: user, email: email_address.email)
```

Comment:
let's not reuse the variable here? maybe inline it or just have separate ones? `identifier_recyled`, `email_changed`